echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineConfigsPollingAgent.groovy

```diff
@@ -176,7 +182,7 @@ class PipelineConfigsPollingAgent extends AbstractPollingAgent {
        */
       List<Pipeline> pipelinesWithNewTriggers = pipelines.findAll {
         def cronTriggers = it.triggers.findAll { it.type == Trigger.Type.CRON.toString() }
-        cronTriggers.any { !actionIds.contains(it.id) }
+        cronTriggers.any { !actionIds.contains(it.id) && !actionIds.find { actionId -> actionId.endsWith(":$it.id") } }
```

Comment:
The composite ids end with a `:` followed by a uuid, so no real chance of a false positive. Feels like the right place to do some cleanup around this would be in scheduled-actions. The two instance action daos have different id naming schemes, so it might make sense to delegate the matching to those classes (or at lease expose the methods to generate the id's so we can use them here).
When we dig back into scheduled-actions to work on that cancellation-relation race condition, maybe we can do more cleanup around this.